sdk/servicebus/tests.yml

```diff
@@ -8,6 +8,7 @@ jobs:
         - name: azure-messaging-servicebus
           groupId: com.azure
           safeName: azuremessagingservicebus
+      TimeoutInMinutes: 120
```

Comment:
Revert timeout. This should not take two hours.

Reply:
Last few test have run for 61 minutes, I will reduce this time to 70 minutes for now.